code/game/gamemodes/wizard/raginmages.dm

```diff
@@ -25,14 +26,13 @@
 	var/obj_count = 1
 	for(var/datum/objective/objective in wizard.objectives)
 		to_chat(wizard.current, "<B>Objective #[obj_count]</B>: [objective.explanation_text]")
-		obj_count++
-	to_chat(wizard.current, "<b>Objective Alpha</b>: Make sure the station pays for its actions against our diplomats")
 	return
 
 /datum/game_mode/wizard/raginmages/check_finished()
 	var/wizards_alive = 0
 	// Accidental pun!
-	var/wizard_cap = (max_mages || (num_players_started() / players_per_mage))
+	var/max_mages = (num_players_started() / players_per_mage)
+	var/wizard_cap = max_mages
 	for(var/datum/mind/wizard in wizards)
```

Comment:
These variables are being declared locally - the ones you have on the datum are not being referenced within here